src/components/filters/input.tsx

```diff
@@ -40,7 +40,8 @@ const InputFilter: FC<Props> = ({
   const inputRef = useRef()
 
   useEffect(() => {
-    ;(inputRef.current as HTMLInputElement).value = initialValue
+    ;(inputRef.current as HTMLInputElement).value =
+      initialValue === undefined ? "" : initialValue
```

Comment:
if we want to also cover in case the value is `null`, we can use the nullish coalescing operator `??`
`;(inputRef.current as HTMLInputElement).value = initialValue ?? ""`

Reply:
Yes, thanks a lot for the suggestion !